code/game/objects/machinery/computer/intel_computer.dm

```diff
@@ -16,12 +16,12 @@
 	///Whether this computer is activated by the event yet
 	var/active = FALSE
 	///How much supply points you get for completing the terminal
-	var/supply_reward = 600
+	var/supply_reward = 1000
 	///How much dropship points you get for completing the terminal
-	var/dropship_reward = 60
+	var/dropship_reward = 100
 
 	///How much progress we get every tick, up to 100
-	var/progress_interval = 1
+	var/progress_interval = 0.75
```

Comment:
this (and max progress being 100) should probs be defines. Not vital to the pr though.

Reply:
not pr vital so I wont do it atleast now, maybe after a tm